### DISCUSSION TITLE
PocketHost 0.5.0 released: backups, metrics, and more

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#1029] [Show and tell]
https://pockethost.io, the open source hosting platform for PocketBase, has reached v0.5.0.

Continued thanks to @ganigeorgiev for all the hard work on PocketBase and for mentioning the PocketHost project when it seems relevant. 

My primary motivation for creating PocketHost continues to be refining the backend for my web game! I'm so far down the rabbit hole developing PocketHost that I'm not sure when I will return to the game code. Everything I do for PocketHost is something I feel would benefit my game and allow me to move the backend off Firebase/Supabase. I feel it's getting VERY close to happening now...but I'm also having so much fun building a hosting platform that it's become a bit distracting ;)

The Svelte frontend is also using rc3 of the JS SDK. The latest realtime features are fantastic, thank you so much for being so quick about those fixes and enhancements @ganigeorgiev!

Onward...

### Metrics

We are now tracking minutes used per month. This gives both us and you a sense of how often your instance is actually running on the platform vs idle time.

Thanks @brewhousedigital for the assist on the cosmetics!

<img width="507" alt="image" src="https://user-images.githubusercontent.com/1068356/201822451-eb19e4d3-bb50-4778-afbf-b1581bf39fec.png">

### Backups

The big new feature that pushed us to 0.5.x was backups. You can now create database backups with the click of a button. This is a zero-downtime operation. Your instance will keep running and the backup snapshot valid as of the time it started. To restore a backup, you have to contact support (for now).

https://user-images.githubusercontent.com/1068356/201821579-f8f0dcef-0b30-44ca-bced-78b74d870df8.mp4

### Support for PocketBase 0.7.10 and 0.8.0-rc2

We do our best to keep pace with PocketBase releases. These are the latest releases of PocketBase.

<img width="381" alt="image" src="https://user-images.githubusercontent.com/1068356/201822640-f45971d2-a21c-4272-9bc8-75823d5b6c5b.png">

### Update to Go 1.19.3

Following suit with PocketBase, our internal Go version was upgraded to 1.19.3 and all binaries were recompiled to take advantage of the latest security improvements.

### Docker build system updates

For anyone wishing to spin up their own hosting service, you might like to know that we have streamlined the Docker experience even more :)

### Display version near PocketHost logo

Credit to: @brewhousedigital 

To make it a little easier for users to track what's going on, we now display the version number by the logo.

<img width="429" alt="image" src="https://user-images.githubusercontent.com/1068356/201821773-fc70266a-08ef-41c4-85c9-46017d16f90d.png">

### Account activation and Password Reset UX enhancements

Credit to: @brewhousedigital 

Account confirmation and password resets now validate through the frontend rather than the pocketbase instance.

<img width="448" alt="image" src="https://user-images.githubusercontent.com/1068356/201822027-dccdbde6-f910-42ff-8d68-b5d8ea02ef7c.png">

### Top Menu UX refresh

Credit to: @brewhousedigital 

Top navigation now makes a little more sense. A `Support` button was added too.

### PocketBase instance version number now shows on dashboard

The PocketBase `platform` and `version` can now be easily found via the dashboard.

<img width="476" alt="image" src="https://user-images.githubusercontent.com/1068356/201822390-2dca040b-1419-41ea-a760-a144edb50cc3.png">

**Comment by Avensen** [comment]
Hi @benallfree, I recently came across PocketHost and wanted to congratulate you for creating such a great service, no doubt something as cool as PocketBase needed something like this. Super easy to use and a beautiful and intuitive interface. Well done.

I wanted to know how I can delete any instance created since I don't see how to do it from the interface (maybe contacting support?).

Also, since I am a Svelte developer myself, I hope I can contribute to the project in some way :)

**Reply by benallfree** [reply]
Hi @Avensen, this discussion board is operated by PocketBase. We have our own support channel over at https://github.com/benallfree/pockethost. 

There is currently not a delete feature, but it is planned in an upcoming release here https://github.com/benallfree/pockethost/issues/35.

In the mean time, you can send an email to ben@pockethost.io with further details and I'll help you.

Looking forward to a Svelte contribution from you! Before you start something, it's best to talk to me just to make sure there aren't any major changes planned or in progress that would conflict with your work.

Thank you for the kind words!
